mkt/urls.py

```diff
@@ -18,6 +18,11 @@
 from mkt.users.views import logout
 
 
+# Hardcore monkeypatching action.
+import jingo.monkey
+jingo.monkey.patch()
```

Comment:
will we need to move the import waffle here? thinking of receiptcheck primarily.